### DISCUSSION TITLE
Help with determining power consumption of epdiy

### DISCUSSION THREAD

**Discussion by milant42** [vroland/epdiy#375] [Q&A] [answered]
Hi all,

This is a begginner question, but I wasn't able to find details elsewhere.

In short - how much does the the screen model affect total power consumption?

For example, I have an epdiy v7 board, and if I have it drive an ED133UT2 will the total power consumption be significantly higher than if I have it drive ED097TC2?

The reason I'm asking is I plan to run it off a battery, and I want to make sure it's enough.

**Comment by martinberlin** [comment, marked as answer]
Hello please find some reports on consumption here:
https://github.com/vroland/epdiy/wiki/Board-v7-Consumption

Moving this to Discussions

**Reply by milant42** [reply]
This is what I needed, thank you!

**Reply by martinberlin** [reply]
Of course it's just a small example. Consumption will ultimately depend on what you do with your Firmware, specially if you use BLE or WiFi, then it ramps up. Espressif MCUs are nice but for using BLE they are not the best option. For example Nordic nRF series use just a small fraction of what S3 uses to make a BLE Beacon
